runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -234,37 +234,74 @@ public static CorfuRuntimeParametersBuilder builder() {
+        // TODO: make it a function of the streaming Queue Size
+
         public static class CorfuRuntimeParametersBuilder extends RuntimeParametersBuilder {
-            int maxWriteSize = Integer.MAX_VALUE;
-            int bulkReadSize = 10;
-            Duration fastLoaderTimeout = Duration.ofMinutes(30);
-            int holeFillRetry = 10;
-            Duration holeFillRetryThreshold = Duration.ofSeconds(1L);
-            Duration holeFillTimeout = Duration.ofSeconds(10);
-            boolean cacheDisabled = false;
-            long maxCacheEntries;
-            long maxCacheWeight;
-            int cacheConcurrencyLevel = 0;
-            long cacheExpiryTime = Long.MAX_VALUE;
-            boolean holeFillingDisabled = false;
-            int writeRetry = 5;
-            int trimRetry = 2;
-            int checkpointRetries = 5;
-            int streamBatchSize = 10;
-            int checkpointReadBatchSize = 5;
-            Duration runtimeGCPeriod = Duration.ofMinutes(20);
-            UUID clusterId = null;
-            int systemDownHandlerTriggerLimit = 20;
-            List<NodeLocator> layoutServers = new ArrayList<>();
-            int invalidateRetry = 5;
+            private int maxWriteSize = Integer.MAX_VALUE;
+            private int bulkReadSize = 10;
+            private Duration fastLoaderTimeout = Duration.ofMinutes(30);
+            private int holeFillRetry = 10;
+            private Duration holeFillRetryThreshold = Duration.ofSeconds(1L);
+            private Duration holeFillTimeout = Duration.ofSeconds(10);
+            private boolean cacheDisabled = false;
+            private long maxCacheEntries;
+            private long maxCacheWeight;
+            private int cacheConcurrencyLevel = 0;
+            private long cacheExpiryTime = Long.MAX_VALUE;
+            private boolean holeFillingDisabled = false;
+            private int writeRetry = 5;
+            private int trimRetry = 2;
+            private int checkpointRetries = 5;
+            private int streamBatchSize = 10;
+            private int checkpointReadBatchSize = 5;
+            private Duration runtimeGCPeriod = Duration.ofMinutes(20);
+            private UUID clusterId = null;
+            private int systemDownHandlerTriggerLimit = 20;
+            private List<NodeLocator> layoutServers = new ArrayList<>();
+            private int invalidateRetry = 5;
             private PriorityLevel priorityLevel = PriorityLevel.NORMAL;
             private Codec.Type codecType = Codec.Type.ZSTD;
             private boolean metricsEnabled = true;
+            private long streamingPollingBlockingTimeMs = 200;
+            private int streamingQueueSize = 60;
+            private int streamingPollingThreadPoolSize = 4;
+            private int streamingNotificationThreadPoolSize = 4;
+            private long streamingNotificationBlockingTimeMs = 200;
+            private int streamingNotificationBatchSize = 20;
```

Comment:
Would be great if these default values here are same as corresponding fields of CorfuRuntime

Reply:
Done.